### PR TITLE
Add progress bar to gallery

### PR DESCRIPTION
**Pull request by willmcgugan**
- Adds progress bar to gallery
- Removes deleted messages

### CODE REVIEW

**Comment by willmcgugan** [comment]
#executivedecision docs update

File: docs/widgets/progress_bar.md [hunk 1]
```diff
@@ -106,27 +106,22 @@ Refer to the [section below](#styling-the-progress-bar) for more information.
...
-
-## Messages
-
-- [ProgressBar.Completed][textual.widgets.ProgressBar.Completed]
-- [ProgressBar.Started][textual.widgets.ProgressBar.Started]
+| Name         | Type    | Default | Description                                                                                             |
+| ------------ | ------- | ------- | ------------------------------------------------------------------------------------------------------- |
+| `percentage` | `float  | None`   | The read-only percentage of progress that has been made. This is `None` if the `total` hasn't been set. |
+| `progress`   | `float` | `0`     | The number of steps of progress already made.                                                           |
+| `total`      | `float  | None`   | The total number of steps that we are keeping track of.                                                 |
```
Comment (rodrigogiraoserrao) [inline docs/widgets/progress_bar.md:113]: Do you have a tool/plugin to format tables like this automatically?

**Review by rodrigogiraoserrao — COMMENTED** [review]
Looks good to me.

File: docs/widgets/progress_bar.md (hunk 1 above)
Comment (willmcgugan) [reply docs/widgets/progress_bar.md:113]: Yes, a vscode extension

**Pull request merged**